<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTitleBar</class>
 <widget class="QWidget" name="CustomTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>40</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CustomTitleBar {
    background-color: #2d2d2d;
    border-bottom: 1px solid #1a1a1a;
}

QPushButton {
    background-color: transparent;
    border: none;
    color: #ffffff;
    font-size: 14px;
    font-weight: bold;
    padding: 8px 12px;
}

QPushButton:hover {
    background-color: #404040;
}

QPushButton#closeButton:hover {
    background-color: #e74c3c;
}

QPushButton#maximizeButton:hover {
    background-color: #3498db;
}

QPushButton#minimizeButton:hover {
    background-color: #f39c12;
}

QLabel#titleLabel {
    color: #ffffff;
    font-size: 14px;
    font-weight: bold;
    padding-left: 10px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>BilibiliPlayer</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="minimizeButton">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>−</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="maximizeButton">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>□</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>